addons/resource.language.en_gb/resources/strings.po

```diff
@@ -6818,7 +6818,18 @@ msgctxt "#13516"
 msgid "Add art"
 msgstr ""
 
-#empty strings from id 13517 to 13549
+#: system/settings/settings.xml
+msgctxt "#13517"
+msgid "Threshold when atempo resampling kicks in"
```

Comment:
Thanks for the explanation. Instead of a "pitch preserving resample algorithm", i would call this feature "pitch correction". So a better title text would be "Threshold for pitch correction".

The help text is still a little confusing to me. Is "resample ratio" just the speed factor, e.g. a 2% resample ratio is a 2% increase/decrease in speed?